### DISCUSSION TITLE
Need an attribute to check code-owner's approval required or not

### DISCUSSION THREAD

**Discussion by rohitgr7** [Mergifyio/mergify#1885] [Q&A]
Hi all,

I am trying to set up auto-update PR on a repo. Is there any condition I can add under the following section to only auto-update if there is no code-owner approval required in a PR?

```
  - name: update PR
    conditions:
      - -conflict
      - -draft # filter-out GH draft PRs
      - "#approved-reviews-by>=3" # number of review approvals
```

Couldn't find anything related to that in the docs https://docs.mergify.io/conditions.html#attributes.

Thanks :)

**Comment by jd** [comment]
Hi @rohitgr7,

GitHub does not expose any information about the code owners feature through their REST API unfortunately.
I just stumbled upon https://github.community/t/how-can-i-query-about-code-owners-for-a-pr/14202/10 and replied, but for the time being we're a bit limited with everything related to code owner, unfortunately.

**Comment by rohitgr7** [comment]
Thanks @jd
